### PR TITLE
Converging British English spellings to American English spellings

### LINKED ISSUE #453: Mixture of British and American English spellings
While reading through, I have come to realise that the document has both British English and American English spellings. After talking to @tariqkurd-repo, we concluded that the document should converge to American English.

### PR DESCRIPTION
**Pull request by francislaus**
This PR is for converging British English spellings to American English spellings. This should close #453

### CODE REVIEW

**Review by arichardson — APPROVED** [review]

**Comment by francislaus** [comment]
@arichardson @tariqkurd-repo This PR should now be ready for review

File: src/riscv-integration.adoc [hunk 1]
```diff
@@ -1033,8 +1033,7 @@ include::img/stvalreg.edn[]
 ==== Supervisor Trap Value Register 2 (stval2)
 
 The <<stval2>> register is an SXLEN-bit read-write register, which is added as
-part of {cheri_base_ext_name} when the implementation supports S-mode. Its CSR
```
Comment (arichardson) [inline src/riscv-integration.adoc:1036]: Did you mean to remove this?

File: src/summary.adoc [hunk 2]
```diff
@@ -38,27 +38,27 @@ This means the following state is added:
...
+Instruction fetch is also authorized by a capability: the program counter capability (<<pcc>>) which extends PC.
 This allows code fetch to be bounded, preventing a wide range of attacks that subvert control flow with integer data.
 Where {cheri_default_ext_name} is supported, the <<pcc>> also contains the <<m_bit,mode bit>> indicating whether the processor is running in integer or capability pointer mode.
 Changing the bounds used for instruction fetch or the pointer mode can be as easy as performing a capability-based jump (<<JALR>> in capability pointer mode).
 <<MODESW_CAP>> and <<MODESW_INT>> instructions are also added to allow cheap mode switching.
 
 Exception codes are added for CHERI-specific exceptions on fetch, jumps, and memory access.
-No other exception paths are added: in particular, capability manipulations do not trap, but may clear the tag on the result capability if the operation is not permitted.
+No other exception paths are added: In particular, capability manipulations do not trap, but may clear the tag on the result capability if the operation is not permitted.
```
Comment (arichardson) [inline src/summary.adoc:61]: I'd keep this lower case

File: src/riscv-integration.adoc [hunk 3]
```diff
@@ -1033,8 +1033,7 @@ include::img/stvalreg.edn[]
...
-address is 0x14b.
+part of {cheri_base_ext_name} when the implementation supports S-mode.
```
Comment (tariqkurd-repo) [inline src/riscv-integration.adoc:1036]:
```suggestion
part of {cheri_base_ext_name} when the implementation supports S-mode. Its CSR
address is 0x14b.
```

File: src/summary.adoc (hunk 2 above)
Comment (tariqkurd-repo) [inline src/summary.adoc:61]:
```suggestion
No other exception paths are added: in particular, capability manipulations do not trap, but may clear the tag on the result capability if the operation is not permitted.
```

File: src/summary.adoc [hunk 4]
```diff
@@ -76,7 +76,7 @@ The added instructions can be split into the following categories:
 
 === Existing Instructions
 
-Existing RISC-V instructions are largely unmodified: in {cheri_int_mode_name}, there is binary compatibility.
+Existing RISC-V instructions are largely unmodified: In {cheri_int_mode_name}, there is binary compatibility.
```
Comment (tariqkurd-repo) [inline src/summary.adoc:79]:
```suggestion
Existing RISC-V instructions are largely unmodified: in {cheri_int_mode_name}, there is binary compatibility.
```

**Review by tariqkurd-repo — APPROVED** [review]

**Pull request merged**